rfcs/0001-notary-integration.md

```diff
@@ -0,0 +1,39 @@
+# Problem
+kpack does not handle image signing.
+
+# Outcome
+kpack integrates with notary to sign images that it builds.
+
+# Actions to take
+There are a few concrete steps to make this work in kpack:
+1. Add a new section to the image config that allows users to configure notary settings:
+  ```
+  notary:
+    host:                <notary server url>
+    rootPrivateKeyRef:   <secret-name>
+    targetPrivateKeyRef: <secret-name>
+  ```
+2. Add a new container after export that uses the notary go client to sign the image. The presence or absence of the notary section in the image config will determine if the notary signing container is executed.
```

Comment:
Does kpack sign both images the tag and the additional timestamp/build-number tag?

Reply:
This impl would sign both the image tag and the build tag.